### PR TITLE
Removing usage of minecraft_data

### PR DESCRIPTION
**Pull request by DaddyWesker**
So, I've removed any usage of minecraft_data using item_list i'm getting from the game. To test this PR you need to use my Vereya code from here https://github.com/trueagi-io/Vereya/pull/18

### CODE REVIEW

File: examples/7_explorer.py [hunk 1]
```diff
@@ -189,9 +189,13 @@ def run(self):
...
     sleep(4)
 
-    # minelogy initialization with current minecraft version
-    mcver = agent.getVersion()
-    mlogy = Minelogy(mcver) # to initialize for usage in this script
+    # initialize minelogy
+    agent.rob.update_in_background()
+    agent.rob.sendCommand('item_list')
+    sleep(2)
+    item_list = agent.rob.mc.observe[0]['item_list']
```
Comment (noskill) [inline examples/7_explorer.py:196]: i would add mc.getItemList method, like for other observations

Comment (DaddyWesker) [reply examples/7_explorer.py]: @noskill  Done. Though, we don't have getRecipe method, for example.

**Review by Necr0x0Der — APPROVED** [review]

**Comment by Necr0x0Der** [comment]
Seems ok, although I'd add a test to `https://github.com/trueagi-io/minecraft-demo/blob/main/tests/vereya/test_observation.py`

**Review by noskill — APPROVED** [review]

**Comment by noskill** [comment]
> Seems ok, although I'd add a test to `https://github.com/trueagi-io/minecraft-demo/blob/main/tests/vereya/test_observation.py`

already there is a test

**Comment by Necr0x0Der** [comment]
> already there is a test

I meant adding a test for `ObservationFromItems` there

**Pull request merged**